vmdb/app/controllers/ems_common.rb

```diff
@@ -219,11 +219,11 @@ def form_field_changed
           # Hide/show C&U credentials tab
           page << "$('metrics_li').#{params[:server_emstype] == "rhevm" ? "show" : "hide"}();"
         end
-        if ["openstack"].include?(params[:server_emstype])
+        if ["openstack", "openstack_infra"].include?(params[:server_emstype])
```

Comment:
That's fair enough - but is that something that should be fixed in this pull request?  It seems like a broader issue.